### ISSUE TITLE
The miner is randomly stopping

### ISSUE THREAD

**Issue by xThomate** [MHDtA-dev/GeoMiner#1]
The miner just randomly stops, it has enough fuel and i dont know what else to try

**Comment by Sarryaz** [comment]
Yeah same for me, I select "Return at start position" but the miner randomly stops during mining. Chunks are force loaded, the miner has enough fuel. I am really frustrated about this. Doesnt always happens, seems to be more reliable in a smaller radius.
